ui/app/AppLayouts/Chat/components/PublicChatPopup.qml

```diff
@@ -0,0 +1,205 @@
+                wrapMode: Text.WordWrap
+                verticalAlignment: Text.AlignTop
+            }
+        }
+
+        Rectangle {
+            id: channelNameBox
+            height: 44
+            color: Theme.grey
+            anchors.top: description.bottom
+            radius: 8
+            anchors.right: parent.right
+            anchors.rightMargin: 16
+            anchors.left: parent.left
+            anchors.leftMargin: 16
+
+            TextField {
+                id: channelName
+                placeholderText: qsTr("chat-name")
+                anchors.left: parent.left
+                anchors.leftMargin: 32
+                anchors.verticalCenter: parent.verticalCenter
+                font.pixelSize: 15
+                background: Rectangle {
+                    color: "#00000000"
+                }
+                width: popup.width - 65
+            }
+
+            Image {
+                id: image4
+                anchors.left: parent.left
+                anchors.leftMargin: 10
+                anchors.verticalCenter: parent.verticalCenter
+                fillMode: Image.PreserveAspectFit
+                source: "../../../img/hash.svg"
+            }
+
+            MouseArea {
+                id: mouseArea
+                anchors.fill: parent
+                onClicked : {
+                    channelName.forceActiveFocus(Qt.MouseFocusReason)
+                }
+            }
+        }
+
+        RowLayout {
+            id: row
+            Layout.fillHeight: false
+            Layout.fillWidth: true
+            anchors.right: parent.right
+            anchors.rightMargin: 65
+            anchors.left: parent.left
+            anchors.leftMargin: 65
+            anchors.top: channelNameBox.bottom
+            anchors.topMargin: 37
+
+            Flow {
+                Layout.fillHeight: false
+                Layout.fillWidth: true
+                spacing: 20
+
+                SuggestedChannel { channel: "ethereum" }
+                SuggestedChannel { channel: "status" }
+                SuggestedChannel { channel: "general" }
+                SuggestedChannel { channel: "dapps" }
+                SuggestedChannel { channel: "crypto" }
+                SuggestedChannel { channel: "introductions" }
+                SuggestedChannel { channel: "tech" }
+                SuggestedChannel { channel: "ama" }
+                SuggestedChannel { channel: "gaming" }
+                SuggestedChannel { channel: "sexychat" }
+                SuggestedChannel { channel: "nsfw" }
+                SuggestedChannel { channel: "science" }
+                SuggestedChannel { channel: "music" }
+                SuggestedChannel { channel: "movies" }
+                SuggestedChannel { channel: "sports" }
+                SuggestedChannel { channel: "politics" }
+            }
+        }
+
+        Rectangle {
+            id: separator2
```

Comment:
Might be worth to create a component for the separator since we are stating to use it more and more